app/controllers/ingestibles_controller.rb

```diff
@@ -28,11 +28,11 @@ def new
   # GET /ingestibles/1/review
   def review
   end
-  
+
   # GET /ingestibles/1/edit
   def edit
     @ingestible.update_parsing # refresh markdown or text buffers if necessary
-    @html = MultiMarkdown.new(@ingestible.markdown).to_html.force_encoding('UTF-8').gsub(/<figcaption>.*?<\/figcaption>/,'') # remove MMD's automatic figcaptions
+    @html = MarkdownToHtml.call(@ingestible.markdown)
```

Comment:
Extracted this code into a service. It would be good to update all places where we use this to use this service instead of copy/pasting code